src/workers/webdav/VirtualDrive.ts

```diff
@@ -328,10 +327,10 @@ const getMacOSMountedInstallerDisks = (): Promise<string[]> => {
   });
 };
 
-const ejectMacOSInstallerDisks = async (): Promise<void> => {
+export const ejectMacOSInstallerDisks = async (): Promise<void> => {
   const installerDisks = await getMacOSMountedInstallerDisks();
   installerDisks.forEach((installerDisk) => {
-    ejectMacOSDisk(installerDisk);
+    ejectMacOSDisk(installerDisk).catch();
```

Comment:
Don't do this please, we are telling the error to shut up, if this fails, it will fail silently, at least log the error. Anyways, isn't better if we fail this operation in case some ejection fails? something like this:
```

for(const installerDisk of installerDisks) {
   await ejectMacOSDisk(installerDisk)
}
```

This way if something crash here, we propagate the error, and we stop the sequence

Reply:
hey, I'm doing it like this because I think it's better to not stop the execution, and keep trying to eject the other disks. Even if none of them are ejected, I think its not something critical enough to propagate the error or stop the execution.